doc/tutorials/sagemaker/sme_deploy_model.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<p>Maybe use bold to highlight that</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>